installation/resources/installer-config-cluster.yaml.tpl

```diff
@@ -70,11 +70,23 @@ data:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: knative-overrides
+  name: knative-serving-overrides
   namespace: kyma-installer
   labels:
     installer: overrides
-    component: knative
+    component: knative-serving
+    kyma-project.io/installation: ""
+data:
+  knative.domainName: "__DOMAIN__"
+---
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Those overrides were for serving. You probably don't need another one for eventing. Same for local config